apps/website/src/components/courses/GroupSwitchModal.tsx

```diff
@@ -167,20 +167,16 @@ const GroupSwitchModal: React.FC<GroupSwitchModalProps> = ({
   const groups = switchingData?.groupsAvailable ?? [];
   const discussions = switchingData?.discussionsAvailable?.[selectedUnitNumber] ?? [];
 
-  const unitOptions = useMemo(() => {
-    if (!courseData?.units) return [];
```

Comment:
I got errors like 'Existing memoization could not be preserved' with this and the useCallbacks below. I couldn't immediately resolve these so I decided to remove the memoization as there was no particular reason to suspect perfomance problems here in the first place